blog/2025-03-14-media-player-toggle-adjusted.md

```diff
@@ -0,0 +1,14 @@
+---
+author: Erik Montnemery
+authorURL: https://github.com/emontnemery
+title: "Changed implementation of media player toggle"
```

Comment:
```suggestion
title: "Media player toggle action changed"
```

Or, if you prefer to keep the current sentence, add `action` at the end: "Changed implementation of media player toggle action"